portfolio/src/main/java/com/google/sps/data/ConvertibleToJSON.java

```diff
@@ -0,0 +1,12 @@
+package com.google.sps.data;
+
+import com.google.gson.Gson;
+
+public abstract class ConvertibleToJSON {
```

Comment:
it is a little bit strange for me to use abstract class here. I think it is more like an interface not abstract class.
BTW, some further reading: https://gist.github.com/kpgalligan/b8c79496abe170807a8e

Actually I think a static function in a helper class is good enough for me. Like public static String convertToJson(Object object).

Reply:
Done - I see. I was trying to avoid having Object type parameter here. Thanks for the reading I will look into it.